pygroups/pygroups/settings/pygroups_secret_variables/base.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
should this be in the repository? (it's a file added by the provision-env script)